packages/reports/tests/acceptance/date-filter-test.js

```diff
@@ -65,4 +65,66 @@ module('Acceptance | date filter', function(hooks) {
       .dom('.filter-values--dimension-date-range-input__high-value .dropdown-date-picker__trigger')
       .includesText('05,', 'The high value is still set after the report is saved');
   });
+
+  test('verify the different time grains work as expected', async function(assert) {
+    await visit('/reports/1/view');
+
+    const timeGrains = ['Hour', 'Day', 'Week', 'Month', 'Quarter', 'Year'];
+
+    for (const grain of timeGrains) {
+      await clickItem('timeGrain', grain);
+      assert.ok(!!$(`.filter-builder__subject:contains(${grain})`)[0], `${grain} is selected`);
```

Comment:
Let's avoid adding more jquery to the codebase where we can. We generally only use jquery for selecting elements that need a `contains` in the selector. Can we replace the other `assert.ok(!!$ ... ` statements with `assert.dom` statements?

Reply:
👍 you can add 
`import findByContains from 'navi-core/test-support/contains-helpers';`
if needed to find the element